### ISSUE TITLE
quotes in delta are not in the 

### ISSUE THREAD

**Issue by pietrop** [quilljs/quill#1664]
First of all thank you, I've found Quill very easy to use and get up and running so far. 

## TL;DR

When using a [delta](https://quilljs.com/guides/designing-the-delta-format/) to get a json representation of the content in the editor the text from `<blockquotes>` elements does not appear to be in the right `insert` object.  It is appended to the previous object. 

### 1. visit [jsfiddle](https://jsfiddle.net/pietrops/w8zvbL99/)

See this [jsfiddle to reproduce the issue](https://jsfiddle.net/pietrops/w8zvbL99/)

This is the HTML snippet to focus on, inside of the editor. that contains some text two quotes, and then some more text. 

```html
ordered the entire fleet to take  a one-day operational pause to  insure that all ships are  running safely.

<blockquote>
First quote
</blockquote>
<blockquote>
Second quote
</blockquote>

 admiral richardson has ordered a  comprehensive review which will  include investigating possible  cyber intrusion and sabotage, although the admiral insis
```

### 2. open dev inspector 
And you'll see the detla as json.

```json
{
  "ops": [
    {
      "insert": "make more than $160,000 in salary and overtime. the agency is now working with congress on a financial fix to better support the president's elite protective force. peter alexander, nbc news, the white house. zbroinchts navy and marine divers are searching for the ten sailors missing after the u.s.s.  john s. mccain collided with an oil tanker off the coast of singapore on sunday. it comes as the navy's top admiral john richardson has ordered the entire fleet to take a one-day operational pause to insure that all ships are running safely.\nFirst quote"
    },
    {
      "attributes": {
        "blockquote": true
      },
      "insert": "\n"
    },
    {
      "insert": "Second quote"
    },
    {
      "attributes": {
        "blockquote": true
      },
      "insert": "\n"
    },
    {
      "insert": "admiral richardson has ordered a comprehensive review which will include investigating possible cyber intrusion and sabotage, although the admiral insists there are \"no indications right now that those are the causes of the incident.\" the u.s.s. john s. mccain is named after senator john mccain's grandfather and father, and in a statement the senator sent his thoughts and prayers to those affected, adding that \"more forceful action is urgently needed to identify and correct the causes of the recent ship collisions.\" this is the fourth incident this year involving a navy vessel in the pacific, and the second time this year a destroyer based in the seven fleet's home port in japan was involved in a collision at sea. incredibly tragic event there.\n"
    }
  ]
}
```

#### **Expected behavior**:

if we take the json for the first quote. we expect `insert` to contain the text of the quote. `First quote` see below:

```json
 {
      "attributes": {
        "blockquote": true
      },
      "insert": "First quote\n"
    },
```

####  **Actual behavior**:

 instead it contains only `\n` and the text of the first quote is appended to the previous element separated by a carriage return char`\n`.

```json
 {
      "attributes": {
        "blockquote": true
      },
      "insert": "\n"
    },
```

#### Other  1

If you try to see how for example a background highlighted is represented, there the element is represented as expected, with the text being contained in the `insert` attribute.

```json 
    {
      "attributes": {
        "background": "#ffc266"
      },
      "insert": "  zbroinchts navy and marine  divers are searching for the ten  sailors missing after the u.s.s.  john s. mccain collided with an  oil tanker off the coast of  singapore on sunday."
    },
```

to reproduce this, highlight so text in the document in the jsfiddle, and click run and inspect the console output. 

#### Other  2

I even tried to save the content of the editor into a temp var

`var tmpContent = quill.getContents();`

_delete the content in the editor_

and then set the content from the temp  var programmatically `quill.setContents(tmpContent)`

and this seem to work fine, so I am assuming the Quill parser for delta, also recognises the block quotes in the current data structure. 

#### **Platforms**:

Include browser, operating system and respective versions

Tried in chrome, but I think is not a browser problem, 

#### **Version**:

`Quill.version` : `"1.3.1"`

## In conclusion 

Let me know if this explanation if clear and whether this issue is by design or what can be done to fix it to meet the expected behaviour.

**Comment by DmitrySkripkin** [comment]
This is how delta works. I don't like it, but here is [explanation](https://quilljs.com/guides/designing-the-delta-format/#line-formatting).

**Comment by pietrop** [comment]
Thanks @DmitrySkripkin, that's helpful, but I still don't think the issue I run into with `blockquote` is by design?

**Comment by jhchen** [comment]
> I still don't think the issue I run into with blockquote is by design?

Why?
